graylog2-server/src/main/java/org/graylog/plugins/views/search/rest/SearchDTO.java

```diff
@@ -41,7 +43,7 @@ public abstract class SearchDTO {
     public abstract String id();
 
     @JsonProperty
-    public abstract Set<QueryDTO> queries();
+    public abstract LinkedHashSet<QueryDTO> queries();
```

Comment:
I understand the usage/creation of the Sets as LinkedHashSets but is it necessary to change the type/interface here?